ietf/community/views.py

```diff
@@ -54,7 +54,7 @@ def view_list(request, email_or_name=None):
     try:
         clist = lookup_community_list(request, email_or_name)
     except MultiplePersonError as err:
-        return HttpResponse(str(err), status=300)
+        raise Http404(str(err))
```

Comment:
We should either 404 with no decoration or use some other response. I'm not quickly coming up with a use-case for telling someone that the string entered matches multiple people is helpful.

Reply:
Yeah! 404 page doesn't show any decorations. I'll remove error strings.